pinot-common/src/test/java/org/apache/pinot/common/data/SchemaTest.java

```diff
@@ -253,6 +253,16 @@ public void testSchemaBuilderAddTime() {
     Assert.assertEquals(schema11, schema12);
   }
 
+  @Test(expectedExceptions = com.fasterxml.jackson.databind.JsonMappingException.class,
+      expectedExceptionsMessageRegExp = "'booleanDimension' field is missing 'dataType' property.*")
+  public void testMissingDataType()
+      throws Exception {
+    URL resourceUrl = getClass().getClassLoader().getResource("missingDataType.schema");
+    Assert.assertNotNull(resourceUrl);
```

Comment:
This isn't an assertion but a precondition. JUnit has the helpful `Assume` for this, but the testng way is to throw a `SkipException` to prevent a confusing assertion error.

Reply:
Done